Formula/arangodb3.rb

```diff
@@ -0,0 +1,85 @@
+class Arangodb3 < Formula
+  desc "The Multi-Model NoSQL Database."
+  homepage "https://www.arangodb.com/"
+  url "https://www.arangodb.com/repositories/Source/ArangoDB-3.0.1.tar.gz"
+  sha256 "f2e9a44bbd73608cace6a3bfe357d2e58aed1eb8b0a4675c36c6a9b15620576a"
+
+  bottle do
+    revision 1
+    sha256 "f9699b28c53f744b7b4701890739d7a9369a6e0b144d486d7fee27ed4b4a9486" => :el_capitan
+    sha256 "7490e2e4026fca6442fce8c6f56d654deb45ba864ca6b7a73872d670b4475d49" => :yosemite
+    sha256 "60d59e09f4a440cffeb077f1de14784519223e6f25ccaaf6df692dc63fd6cac8" => :mavericks
+  end
+
+  head do
+    url "https://github.com/arangodb/arangodb.git", :branch => "unstable"
+    depends_on "cmake" => :build
+  end
```

Comment:
The depends_on "cmake" does not need to be limited to any particular spec. So this simplifies to
```
head do
  url "https://github.com/arangodb/arangodb.git", :branch => "unstable"
end
```
But when the only element in a head block is the url, it should be simplified to the non-block form
```
head "https://github.com/arangodb/arangodb.git", :branch => "unstable"
```
Audit will require that to be placed below the checksum like this:
```
sha256 "f2e9a44bbd73608cace6a3bfe357d2e58aed1eb8b0a4675c36c6a9b15620576a"
head "https://github.com/arangodb/arangodb.git", :branch => "unstable"
```